internal/driver/virtualdriver.go

```diff
@@ -198,3 +198,12 @@ func deleteVirtualResources(driver *VirtualDriver, deviceName string) error {
 		return nil
 	}
 }
+
+func (d *VirtualDriver) Discover() error {
+	return fmt.Errorf("Driver's Discover function isn't implemented")
```

Comment:
please use lowercase for error

Reply:
Opened PR to update the error message https://github.com/edgexfoundry/device-virtual-go/pull/380